Automattic/wp-calypso
# Performance Profiler: Pass locale to LLM Query

Resolves https://github.com/Automattic/dotcom-forge/issues/10051

## Proposed Changes

* Adds an additional `locale` body param to the `/odie/assistant/performance-profiler` endpoint

## Why are these changes being made?


* To return the LLM responses translated to the locale configured by the user

## Testing Instructions



* Apply the following diff D168288-code and sandbox the API
* Log-in with a user configured to use a `locale` different than `EN`
* Apply this branch and run a Speed Test or open an existing report, e.g. `/speed-test-tool?hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzA3MX0.m3-63iGZmpJOpOfkZ4ww184EoKd8P3UHzTYwv1VLQUo&url=https%3A%2F%2Fwordpress.com%2F&tab=mobile`
* Scroll down to an insight and expand the result
* Make sure both, the AI-generated response, and the insight are correctly translated to the `locale` configured in step 2

## Pre-merge Checklist
